tests/test_quixstreams/fixtures.py

```diff
@@ -364,6 +360,16 @@ def factory(workspace_id: Optional[str] = None):
         cfg_builder.get_topic.side_effect = lambda topic: {
             "id": cfg_builder.prepend_workspace_id(topic)
         }
+
+        connection = ConnectionConfig(bootstrap_servers=kafka_container.broker_address)
+        cfg_builder._librdkafka_connection_config = connection
+        cfg_builder.librdkafka_connection_config = connection
```

Comment:
Do we need to patch the `_librdkafka_connection_config` here too?
Can we somehow patch only the public method?